src/common/lib/python/common_extension.cc

```diff
@@ -693,6 +703,23 @@ static PyObject *PyTask_execution_dependencies_string(PyTask *self) {
                                    fbb.GetSize());
 }
 
+static PyObject *PyTask_to_serialized_flatbuf(PyTask *self) {
+  RAY_CHECK(use_raylet(self));
+
+  const std::vector<ObjectID> execution_dependencies(
+      *self->execution_dependencies);
+  auto const execution_spec = ray::raylet::TaskExecutionSpecification(
+      std::move(execution_dependencies));
```

Comment:
This would be equivalent to saying
```
auto const execution_spec = ray::raylet::TaskExecutionSpecification(
      *self->execution_dependencies);
```
without creating a temporary `execution_dependencies`, why not that way?

Reply:
Hmm, thinking about it more, that might give a compile error and TaskExecutionSpecification would need a non-move version for this to work.